metadatastore/commands.py

```diff
@@ -169,11 +326,9 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
-    beamline_config = _get_mongo_document(beamline_config, BeamlineConfig)
     run_start = RunStart(time=time, scan_id=scan_id,
-                         time_as_datetime=_todatetime(time), uid=uid,
```

Comment:
However, having datetimes in the non-event documents makes your life so so so much better whenever you need to drop to a mongo shell and debug